cpp/autosar/test/rules/A16-2-2/test.cpp.clang

```diff
@@ -0,0 +1,6 @@
+#include "test.hpp" //NON_COMPLIANT
+#include <string>   //COMPLIANT
+#include <vector>   //COMPLIANT
+#include <vector>   //COMPLIANT
```

Comment:
This line is duplicated. I would also like to see a "non_compliant" std library include, if we could find one.